python/tvm/relax/frontend/torch/fx_translator.py

```diff
@@ -476,6 +476,11 @@ def _inplace_fill(self, node: fx.Node) -> relax.Var:
         self.env[node.args[0]] = filled
         return filled
 
+    def _inplace_copy(self, node: fx.Node) -> relax.Var:
+        dst = self.env[node.args[1]]
```

Comment:
According to the official doc, the first arg is src. The logic seems correct but the naming of the intermediate variable is misleading.
https://pytorch.org/docs/stable/generated/torch.Tensor.copy_.html